### DISCUSSION TITLE
edge functions local development json import

### DISCUSSION THREAD

**Discussion by chirastefan** [supabase/supabase#8825] [Questions] [answered]
Hello, I'm running supabase locally with docker and serving a function with `supabase functions serve myFunction`. I'm trying to import a JSON file, but the file is not recognized (Intellij doesn't help either) even though the path is correct.

<img width="553" alt="Screen Shot 2022-09-07 at 12 21 25" src="https://user-images.githubusercontent.com/16649178/188842191-b9a26e1f-f3a5-4881-857a-46aafb6de4e1.png">
<img width="972" alt="Screen Shot 2022-09-07 at 12 18 17" src="https://user-images.githubusercontent.com/16649178/188842201-4e0d34cc-0be8-472c-aacd-ea12fe866d80.png">

**Comment by laktek** [comment, marked as answer]
@chirastefan Can you move `config.json` under `functions` directory?

**Reply by chirastefan** [reply]
Hi, I don't need the edge functions anymore, but if I ever do and need this, I'll try this, ty.
